### PR TITLE
AC-330 : Makes register form more user friendly

### PR DESCRIPTION
**Pull request by DefCon-007**

### CODE REVIEW

File: openmrs-client/src/main/res/values/states.xml [hunk 1]
```diff
@@ -0,0 +1,4612 @@
...
+        <item>Pondicherry</item>
+        <item>Punjab</item>
+        <item>Rajasthan</item>
+        <item>Sikkim</item>
+        <item>Tamil Nadu</item>
+        <item>Telangana</item>
+        <item>Tripura</item>
+        <item>Uttar Pradesh</item>
+        <item>Uttarakhand</item>
+        <item>Vaishali</item>
```
Comment (AvijitGhosh82) [inline openmrs-client/src/main/res/values/states.xml]: Vaishali? This doesn't look right...what is the source of your data?

File: openmrs-client/src/main/res/values/states.xml [hunk 2]
```diff
@@ -0,0 +1,4612 @@
...
+        <item>Calabria</item>
+        <item>Campania</item>
+        <item>Cartoceto</item>
+        <item>Caserta</item>
+        <item>Catania</item>
+        <item>Chieti</item>
+        <item>Como</item>
+        <item>Cosenza</item>
+        <item>Cremona</item>
+        <item>Cuneo</
```
Comment (AvijitGhosh82) [inline openmrs-client/src/main/res/values/states.xml]: Please correct typos and other such errors in data source, @DefCon-007

File: openmrs-client/src/main/res/values/states.xml (hunk 1 above)
Comment (DefCon-007) [inline openmrs-client/src/main/res/values/states.xml]: I got the data from a GitHub Repository. Will find an alternate source.

**Comment by DefCon-007** [comment]
@AvijitGhosh82  : Made the necessary changes.

**Pull request merged**